muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/chat/ChatRESTService.java

```diff
@@ -160,27 +168,44 @@ public Response chatStatus() {
       return Response.status(Status.FORBIDDEN).entity("Must be logged in").build();
     }
 
+    SchoolDataIdentifier identifier = sessionController.getLoggedUser();
+    
+    if (identifier == null) {
+        return Response.status(Status.INTERNAL_SERVER_ERROR).entity("Couldn't find logged user").build();
+      }
+	  
     String enabledUsersCsv = pluginSettingsController.getPluginSetting("chat", "enabledUsers");
+    
     if (enabledUsersCsv == null) {
-      enabledUsersCsv = "";
+      return null;
```

Comment:
Probably means that the chat is not configured? Likely still better to return something, maybe `Response.status(Status.INTERNAL_SERVER_ERROR).build()` or `Status.NOT_IMPLEMENTED` etc. w/e makes most sense.